mender-update/daemon/context.cpp

```diff
@@ -142,6 +146,299 @@ Context::Context(main_context::MenderContext &mender_context, events::EventLoop
 	download_client(http::ClientConfig(mender_context.GetConfig().server_url), event_loop) {
 }
 
+static string GenerateStateDataJson(const StateData &state_data) {
+	string content = "{";
```

Comment:
I've been using `stringstream` for this because `ss << something` saves some parentheses and makes things easier to read for me. But YMMV. I have no clue about performance differences.

Reply:
I don't know either, but I agree it looks better. Fixed, and I've tried to move-return it, so there won't an additional copy.